backend/api/adjudicationsApi.ts

```diff
@@ -0,0 +1,48 @@
+import { OauthApiClient } from './oauthEnabledClient'
+import contextProperties from '../contextProperties'
+import { mapToQueryString } from '../utils'
+
+export const adjudicationsApiFactory = (client: OauthApiClient) => {
+  const processResponse = (context) => (response) => {
+    contextProperties.setResponsePagination(context, response.headers)
+    return response.body
+  }
+
+  const get = (context, url, resultsLimit?, retryOverride?) =>
+    client.get(context, url, { resultsLimit, retryOverride }).then(processResponse(context))
+
+  const getAdjudications = async (context, offenderNumber, params, pageOffset, pageLimit) => {
+    const paramsWithPageAndSize = {
+      ...params,
+      page: pageOffset / pageLimit,
+      size: pageLimit,
+    }
+
+    const response = await get(
+      context,
+      `/adjudications/${offenderNumber}/adjudications${params && `?${mapToQueryString(paramsWithPageAndSize)}`}`
+    )
+
+    return {
+      ...response,
+      pagination: {
+        pageOffset: response.results.pageable.offset,
+        pageLimit: response.results.pageable.pageSize,
+        totalRecords: response.results.totalElements,
+      },
+    }
+  }
+
+  const getAdjudicationDetails = (context, offenderNumber, adjudicationNumber) =>
+    get(context, `/adjudications/${offenderNumber}/charge/${adjudicationNumber}`)
+
+  const getAdjudicationsForBooking = (context, bookingId) => get(context, `/adjudications/by-booking-id/${bookingId}`)
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L102-L140) in the AdjudicationsAPI